website/docs/vscode.md

```diff
@@ -6,231 +6,457 @@ sidebar_label: Visual Studio Code
-configurations: stable, beta, and experimental. Workspaces can specify
-alternative configurations via the `sorbet.lspConfigs` setting.
+### `>Sorbet: Toggle highlighting untyped code`
 
-Users can select between these configurations on-the-fly by clicking on "Sorbet"
-in VS Code's status line and selecting "Configure Sorbet". Sorbet will then
-restart in the chosen configuration. Sorbet will also remember this
-configuration choice for the user's future sessions in the workspace.
+[`>Sorbet: Toggle highlighting untyped code`]:
+  #sorbet-toggle-highlighting-untyped-code
 
-## Disabling the Sorbet extension
+See [Highlighting untyped code](highlight-untyped.md)
 
-There are multiple ways to disable the Sorbet extension depending on your goals.
+### `` >Sorbet: Toggle the autocomplete nudge in `typed: false` files ``
 
-- You can click on Sorbet in VS Code's status line and click on _Disable
-  Sorbet_, which will immediately stop Sorbet in that workspace. You will need
-  to click on _Enable Sorbet_ to reenable Sorbet in that workspace. This is a
-  handy way to temporarily disable Sorbet if it is causing problems.
-- Workspaces can set the `sorbet.enabled` setting to `false`, which prevents
-  Sorbet from running in the workspace.
+[``>Sorbet: Toggle the autocomplete nudge in `typed: false` files``]:
+  #sorbet-toggle-the-autocomplete-nudge-in-typed-false-files
 
-## Troubleshooting and FAQ
+By default, Sorbet shows a completion item in `# typed: false` files when there
+are no completion results due to the file being `# typed: false`, like this:
 
-### Startup
+<img src="/img/typed-false-nudge.png" style="max-width: 463px; margin-left: 0;" />
 
-#### Error: "Sorbet's language server requires a single input directory. However, 0 are configured"
+The goal of the nudge is to encourage upgrading the current file
+[from `# typed: false` to `# typed: true`](static.md). Use this command to
+disable the nudge.
 
-This error can happen if you have not initialized Sorbet in your project. Please
-[follow the instructions](adopting#step-2-initialize-sorbet-in-our-project) to
-initialize Sorbet.
+As a reminder, most features, including autocompletion,
+[have degraded support](lsp-typed-level.md) in `# typed: false` files.
 
-If initializing Sorbet in your project is not desirable or possible, an
-alternative fix is to override the default extension configuration in the
-project's `.vscode/settings.json` file and provide the project directory as
-`"."`:
+## Configuration Reference
+
+This section documents configuration options which live in the
+[User or Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings)
+of VS Code.
+
+For a complete reference of the Sorbet extension's configuration options, their
+types, and their defaults, see the `"configuration"` section of the
+[package.json](https://github.com/sorbet/sorbet/blob/master/vscode_extension/package.json)
+file.
+
+### Sticky configuration
```

Comment:
For posterity, there's a larger discussion of this in Stripe's internal issue tracker here:

https://go/RKSKuCsK